src/m1_facilitator/models/Gateway.ts

```diff
@@ -0,0 +1,107 @@
+
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+
+import BigNumber from 'bignumber.js';
+import Comparable from '../../m0_facilitator/observer/Comparable';
+
+/**
+ * Type of gateways.
+ */
+export enum GatewayType {
+  Consensus = 'consensus',
+  Most = 'most',
+  ERC20 = 'erc20',
+  NFT = 'nft'
+}
+
+/**
+ * Represents Gateway model object.
+ */
+export default class Gateway extends Comparable<Gateway> {
+
```

Comment:
Block must not be padded by blank lines.

Reply:
Done